pkg/build/build.go

```diff
@@ -1,7 +1,8 @@
 package build
 
 var (
-	Name    = "nixos-facter"
-	Version = "v0.0.0+dev"
-	System  = ""
+	Name          = "nixos-facter"
+	Version       = "v0.0.0+dev"
+	System        = ""
+	ReportVersion = "0"
```

Comment:
```suggestion
	ReportVersion = "1"
```
I would keep that version tracking in the Go code. The nature of that number is a bit different than the release since that can be bumped in the same commit that changes the schema.

Reply:
So, don't set this value via `-X`, just manage it directly by bumping the value in the go code directly when required?

Reply:
Yes, that collapses a bit of complexity.